grading/main_configure.cpp

```diff
@@ -220,9 +221,22 @@ nlohmann::json validate_notebook(const nlohmann::json& notebook, const nlohmann:
                       << (item_label.empty() ? "[no label]" : item_label) << std::endl;
           }*/
         }
+        // Check if the indices mapped to users are in 'from_pool' array range
+        for (auto i=user_item_map.begin(); i!=user_item_map.end(); i++) {
+           int item_index = i.value();
+           if(item_index < 0 || item_index > in_notebook_cell_from_pool.size()) {
```

Comment:
Is notebook indexing based on 0 indexing, I think `item_index` should be >= `in_notebook_cell_from_pool.size()` in this case since I believe the rest of system assumes 0 is the first index in a notebook pool

Reply:
Yeah, I missed `=` in there, thanks!